modules/config.js

```diff
@@ -7,6 +7,7 @@ var ExecutionEnvironment = require('exenv');
 var _matchMediaFunction = ExecutionEnvironment.canUseDOM &&
   window &&
   window.matchMedia;
+var _mediaQueryAliases = new Map();
```

Comment:
I'm not sure we should be using `Map`s in the Radium core yet. They require more polyfill that what is needed for just React. So far we've only required the same ES5 polyfills, and let Babel do the rest.